docs/modules/jquery.md

```diff
@@ -0,0 +1,17 @@
+# jQuery
+
+jQuery is a great project that at one time greatly simplified the life of web developers. However, now almost all features from this project can be easily replaced with native JavaScript methods and it makes no sense to load this quite a big dependence.
```

Comment:
i would reword the last bit as:

> so it makes little sense to load a dependency this large to provide them

seems like `dependence` should be `dependency` at least

Reply:
Ok. Updated.